spec/models/users_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Users do
+  subject { Users.new(name: 'John Connor', photo: 'www.google.com', bio: 'Hello my name is John') }
+
+  before { subject.save }
+
+  it 'name should be John Connor' do
+    subject.name = 'John Connor'
+  end
+
+  it 'photo should be www.google.com' do
+    subject.photo = 'www.google.com'
+  end
+
+  it 'bio should be Hello my name is John' do
+    subject.bio = 'Hello my name is John'
```

Comment:
* According to the project [requirements ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods%20and%20validations.) 

> Add unit specs for all of your models' methods and validations.
* Kindly add unit specs for the `lastest` method 
* And write unit specs that cover all validations specified here 🙏